ibmpisession/logger.go

```diff
@@ -24,10 +24,10 @@ func (IBMPILogger) Debugf(format string, args ...interface{}) {
 	if len(format) == 0 || format[len(format)-1] != '\n' {
 		format += "\n"
 	}
-	fmt.Fprintf(os.Stderr, format, sanatizeArgs(args)...)
+	fmt.Fprintf(os.Stderr, format, SanatizeArgs(args)...)
```

Comment:
Want is the need here?

Reply:
I made this uppercase so it can be an exported function and it can be used in the testing directory.